static/js/util.ts

```diff
@@ -239,23 +249,23 @@ export function get_edit_event_topic(obj) {
     return obj.topic;
 }
 
-export function get_edit_event_orig_topic(obj) {
+export function get_edit_event_orig_topic(obj: UpdateMessageEvent): string | undefined {
     return obj.orig_subject;
 }
 
-export function is_topic_synonym(operator) {
+export function is_topic_synonym(operator: string): boolean {
     return operator === "subject";
 }
 
-export function convert_message_topic(message) {
+export function convert_message_topic(message: Message): void {
     if (message.topic === undefined) {
         message.topic = message.subject;
     }
 }
 
-export function clean_user_content_links(html) {
+export function clean_user_content_links(html: string): string {
     const content = new DOMParser().parseFromString(html, "text/html").body;
-    for (const elt of content.querySelectorAll("a")) {
+    for (const elt of Array.from(content.querySelectorAll("a"))) {
```

Comment:
Unnecessary; `querySelectorAll` is already iterable. (It wasn’t in ancient browsers, but we’ve been polyfilling for that and don’t care anymore.)